### ISSUE TITLE
When uploading chunks of a file (we gather all the tasks). If one fails then it is added to the bottom of the queue. Ideally it should be put at the beginning of the queue to be picked up as soon as possible.

### ISSUE THREAD

**Issue by GitHK** [ITISFoundation/osparc-issues#974]
When uploading a file to S3, we divided it in small chunks and try to upload them in parallel. Since the chinks are small, you end up with lots of these chunks. You cannot launch 1000 parallel connections on a machine (would cause issues), you usually have a few workers uploading and them. All operations are stored in a queue. Now when we have a fail during one of these chunk uplaods, the "retry" is added at the bottom of the queue which is not efficient.

TL;DR:
When nodeports retries chunk upload operations, these end up being picked up with lower priority than other existing chunks, this should be changed.
